README.md

```diff
@@ -95,6 +95,10 @@ in various programming languages.
   by Andrej Bauer and Danel Ahman  
   ([www](https://github.com/andrejbauer/coop))
 
+* **Unison** a programming language with effects called abilities, inspired by Frank  
+  ([www](https://www.unisonweb.org/docs/language-reference#abilities-and-ability-handlers))  
```

Comment:
Please remove the two trailing whitespaces on this line to make this item render like the other items.
```suggestion
  ([www](https://www.unisonweb.org/docs/language-reference#abilities-and-ability-handlers))
```

Reply:
For consistency we ought to add an author too, I suppose you can add `by Unison Computing  `.

Reply:
Thank you for the review :)
Removed spaces so www and Github links will be in one line.
Added author and since you want consistency I added author to Idris (I added Idris some time ago so I feel responsible).

Reply:
Thanks!